docs/source/user-guide/concepts/packages.rst

```diff
@@ -0,0 +1,93 @@
+==============
+Conda packages
+==============
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. _concept-conda-package:
+
+What is a conda package?
+========================
+
+A conda package is a compressed tarball file (.tar.bz2) that contains:
+
+* system-level libraries
+* Python or other modules
+* executable programs and other components
+* metadata under the ``info/`` directory
+* a collection of files that are installed directly into an ``install`` prefix
+
+Conda keeps track of the dependencies between packages and platforms.
+The conda package format is identical across platforms and
+operating systems.
+
+Only files, including symbolic links, are part of a conda
+package. Directories are not included. Directories are created
+and removed as needed, but you cannot create an empty directory
+from the tar archive directly.
+
+Using packages
+==============
+
+* You may search for packages
+
+.. code-block:: bash
+
+  conda search scipy
+
+
+* You may install a package
+
+.. code-block:: bash
+
+  conda install scipy
+
+
+* You may build a package
```

Comment:
It might be useful to point out that users need to install conda build in order to do that. Maybe a link to conda build docs would be good too?